#!/usr/bin/env python
PACKAGE = "pantilt_position_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pan_angular", int_t, 0 , "goalset", 90, 0, 255 )
gen.add("tilt_angular", int_t, 0 , "goalset", 104, 0, 255 )
gen.add("b", bool_t, 0 , "parameter", True )
gen.add("distance_x", double_t, 0, "target_positionx", 1, 0, 10 )
gen.add("distance_y", double_t, 0, "target_positiony", 0, -5,  5)
gen.add("distance_z", double_t, 0, "target_positionz", 30, 0, 8.66 )

exit(gen.generate(PACKAGE, "pantilt_position_control","ang_vel_param"))
